runtime/compilers/rillv1/parse_metrics_view.go

```diff
@@ -238,12 +239,6 @@ func (p *Parser) parseMetricsView(ctx context.Context, node *Node) error {
 			return fmt.Errorf("found duplicate dimension or measure name %q", dim.Name)
 		}
 		names[lower] = true
-
-		lower = strings.ToLower(dim.Column)
-		if ok := columns[lower]; ok {
-			return fmt.Errorf("found duplicate dimension column name %q", dim.Column)
-		}
-		columns[lower] = true
```

Comment:
If removing this, should also remove the `columns := make(map[string]bool)` on line 216 and other refs to it

Reply:
Ya removing them. It just didnt make sense anymore since we do not check duplicate of expression.